filecoin-proofs/src/types/porep_config.rs

```diff
@@ -54,6 +55,22 @@ impl From<PoRepConfig> for SectorSize {
 }
 
 impl PoRepConfig {
+    /// construct PoRepConfig by groth16
+    pub fn new_groth16(sector_size: u64, porep_id: [u8; 32], api_version: ApiVersion) -> Self {
```

Comment:
Until additional proof systems are supported in `rust-fil-proofs`, I think that it's better to exclude the "groth16` identifier from the function name as Groth16 is the only proof system currently supported. If/when additional proof systems are supported, I definitely think that it makes sense to use the function name `new_groth16`, but for now, I think that `new` is a better name.